puppet/modules/leap/manifests/logfile.pp

```diff
@@ -0,0 +1,26 @@
+#
+# make syslog log to a particular file for a particular process.
+#
+
+define leap::logfile($process=$title) {
+  $logfile = "/var/log/leap/${title}.log"
```

Comment:
the log file should be named after the title, not the process, which might be weird.